src/openrct2/ride/Vehicle.h

```diff
@@ -530,7 +530,9 @@ enum : uint32_t
     VEHICLE_UPDATE_FLAG_USE_INVERTED_SPRITES = (1 << 11), // Used on rides where trains can run for extended periods of time,
                                                           // i.e. the Flying, Lay-down and Multi-dimension RCs.
     VEHICLE_UPDATE_FLAG_12 = (1 << 12),
-    VEHICLE_UPDATE_FLAG_ROTATION_OFF_WILD_MOUSE = (1 << 13) // After passing a rotation toggle track piece this will enable
+    VEHICLE_UPDATE_FLAG_ROTATION_OFF_WILD_MOUSE = (1 << 13), // After passing a rotation toggle track piece this will enable
+    VEHICLE_UPDATE_FLAG_SINGLE_CAR = (1 << 14), // New flag used to override UpdateMotion to apply only to an individual vehicle
+                                                // on a train
```

Comment:
```suggestion
    VEHICLE_UPDATE_FLAG_SINGLE_CAR = (1 << 14), // Flag to override UpdateMotion to apply only to an individual vehicle on a train
```

I guess the `new` will just become outdated overtime, so I'd just simplify this.

Reply:
Or specify that _we_ added it and didn't inherit it from RCT2.

Reply:
Changed it to "OpenRCT2 Flag: Used to...."